ignite/services/network/node.go

```diff
@@ -0,0 +1,87 @@
+package network
+
+import (
+	"context"
+	"encoding/base64"
+
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	spntypes "github.com/tendermint/spn/pkg/types"
+
+	"github.com/ignite-hq/cli/ignite/pkg/cosmosclient"
+)
+
+type (
+	// IBC is node client info.
+	IBC struct {
+		ConsensusState spntypes.ConsensusState
+		ValidatorSet   spntypes.ValidatorSet
+		UnbondingTime  int64
+		Height         uint64
+	}
+
+	// Node is node builder.
+	Node struct {
+		cosmos       CosmosClient
+		stakingQuery stakingtypes.QueryClient
+	}
+)
+
+func NewNode(ctx context.Context, nodeAPI string) (*Node, error) {
+	c, err := cosmosclient.New(ctx, cosmosclient.WithNodeAddress(nodeAPI))
```

Comment:
We should get the client from outside for easy testing, like we do in network.New